### DISCUSSION TITLE
Tabs in Item Detail page forms

### DISCUSSION THREAD

**Discussion by mdocter** [directus/directus#3045] [Feature Requests]
It would be great to be able to use tabs in large forms to prevent very long forms in case of many fields.

**Comment by benhaynes** [comment]
This will likely be built as a "Group" interface (alias). We're working on adding this, but will likely only have a basic grouping interface for 9.0.0... with new ones (like this one) coming later.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by hanneskuettner** [comment]
But you can check out and install my extension that implements this interface https://github.com/hanneskuettner/directus-extension-group-tabs-interface
